public/r2/v1/swagger/swagger.json

```diff
@@ -863,20 +936,80 @@
                     "type": "integer",
                     "format": "unixMillis"
                 },
-                "mappingRules": {
+                "mappingRulesChanges": {
                     "type": "array",
                     "items": {
                         "$ref": "#/definitions/MappingRule"
                     }
                 },
-                "rollupRules": {
+                "rollupRulesChanges": {
```

Comment:
I think these should be rollup rules instead of rollup rule changes?

Reply:
I should have caught this. I think this was bad find and replace.